internal/config/rtfiles.go

```diff
@@ -40,15 +40,12 @@ var allFiles [][]RuntimeFile
 var realFiles [][]RuntimeFile
 
 func init() {
```

Comment:
nit: This `init()` seems not needed now. `InitRuntimeFiles()` is called during early startup anyway.

Reply:
Done, but the tests and possible future usages need to call these functions explicit since no further implicit initialization takes place.